addons/remaining-replies/userscript.js

```diff
@@ -0,0 +1,54 @@
+export default async function ({ addon, msg }) {
+  function addRemainingReplyCount(comment) {
+    // skip the main comment input at the top of the page
+    if (comment.classList.contains("compose-row")) return;
+    // commenting was turned off for the studio
+    if (!comment.querySelector(".comment-reply span")) return;
+
+    let parentComment = comment.parentElement.classList.contains("replies")
+      ? comment.parentElement.parentElement.children[0]
+      : comment;
+
+    const parentCommentID = parentComment.getAttribute("id");
+    const parentCommentData = addon.tab.redux.state.comments.comments.filter(
+      // substring in order to remove the "comments-" prefix
+      (comment_in_list) => comment_in_list.id == parentCommentID.substring(9)
+    )[0];
+
+    const remainingReplies = 25 - parentCommentData?.reply_count;
+
+    const span = document.createElement("span");
+    span.classList.add("sa-replies-remaining");
+    if (remainingReplies > 10) span.classList.add("sa-replies-remaining-hide");
+    span.style.marginLeft = "0.25em";
```

Comment:
Would it be better to include a leading space in the message?

Reply:
Probably not, as the leading space may get ignored by translators.